From aec5a60d582ebdab2d3707137e0ed7fa9668d484 Mon Sep 17 00:00:00 2001
From: Sagi Maimon <smaimon@habana.ai>
Date: Thu, 25 Feb 2021 12:27:53 +0200
Subject: [PATCH] [SW-32460] gaudi: security: add fake sec in build

Add capability to build preboot in fake security mode.
Add to makefile the activation of pre-processor
symbol FAKE_SECURITY.
All security fake code is defined using the pre-processor
symbol FAKE_SECURITY.

Change-Id: Iea0ffa97ad81dc71f9b16e3a8db5b4375f8ebea4
Signed-off-by: Sagi Maimon <smaimon@habana.ai>
---

diff --git a/Makefile b/Makefile
index 63c9554..20d3ad3 100644
--- a/Makefile
+++ b/Makefile
@@ -13,6 +13,10 @@
 	CFLAGS += -DTARGET_IS_PLDM
 endif
 
+ifeq ($(FAKE_SECURITY),ON)
+	CFLAGS += -DFAKE_SECURITY
+endif
+
 CFLAGS += -Os -fdata-sections -ffunction-sections
 SFLAGS := $(CFLAGS) -D__ASSEMBLY__
 
